7.analyze-coverage/scripts/nbconverted/visualize_coverage.r

```diff
@@ -410,8 +410,8 @@ tcga_cm_label <- cowplot::ggdraw() +
                         angle = 90)
 
 label_main <- cowplot::plot_grid(tcga_tf_label,
-                                 tcga_cm_label,
                                  tcga_react_label,
+                                 tcga_cm_label,
```

Comment:
Not seeing a difference in the figure axes.  What changed?

Reply:
The bottom left x axis was not labeled